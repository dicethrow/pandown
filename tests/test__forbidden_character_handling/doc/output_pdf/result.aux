\relax 
\providecommand\hyper@newdestlabel[2]{}
\abx@aux@refcontext{nty/global//global/global}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\providecommand \oddpage@label [2]{}
\@writefile{toc}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lof}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lot}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces alt text\relax }}{2}{figure.caption.1}\protected@file@percent }
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {part}{I\hspace  {1em}This is part one}{2}{part.1}\protected@file@percent }
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{\numberline {1}heading one in /main.md}{2}{section.1}\protected@file@percent }
\newlabel{heading-one-in-main.md}{{1}{2}{heading one in /main.md}{section.1}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {part}{II\hspace  {1em}This is part two}{2}{part.2}\protected@file@percent }
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{\numberline {2}To do}{2}{section.2}\protected@file@percent }
\newlabel{to-do}{{2}{2}{To do}{section.2}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{\numberline {3}heading level 2 in the .md}{2}{section.3}\protected@file@percent }
\newlabel{heading-level-2-in-the-.md}{{3}{2}{heading level 2 in the .md}{section.3}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}heading level 3 in the .md}{2}{subsection.3.1}\protected@file@percent }
\newlabel{heading-level-3-in-the-.md}{{3.1}{2}{heading level 3 in the .md}{subsection.3.1}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{\numberline {4}Heading level 1 (should become 2) in content/a\_outerfolder/main.md}{3}{section.4}\protected@file@percent }
\newlabel{heading-level-1-should-become-2-in-contenta_outerfoldermain.md}{{4}{3}{Heading level 1 (should become 2) in content/a\_outerfolder/main.md}{section.4}{}}
