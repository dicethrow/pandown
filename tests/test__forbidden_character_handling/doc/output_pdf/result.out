\BOOKMARK [0][-]{part.1}{I This is part one}{}% 1
\BOOKMARK [1][-]{section.1}{heading one in /main.md}{part.1}% 2
\BOOKMARK [0][-]{part.2}{II This is part two}{}% 3
\BOOKMARK [1][-]{section.2}{To do}{part.2}% 4
\BOOKMARK [1][-]{section.3}{heading level 2 in the .md}{part.2}% 5
\BOOKMARK [2][-]{subsection.3.1}{heading level 3 in the .md}{section.3}% 6
\BOOKMARK [1][-]{section.4}{Heading level 1 \(should become 2\) in content/a\137outerfolder/main.md}{part.2}% 7
